projects/hathor-wallet-headless/api-docs-ci.md

```diff
@@ -0,0 +1,63 @@
+# Summary
+This feature automates the process of uploading the updated API Docs for the Headless Wallet, while giving the developer a preview of the contents, error checking and linting.
+
+A script is made available for the developer to make all the necessary checks locally and a GitHub Workflow is implemented to:
+- Validate and lint the `api-docs.js` file on every PR to the `dev` branch
+- Deploy the updated docs on every new successful release
+
+# Motivation
+Currently this upload is made manually by a team member, extracting data from the `src/api-docs.js` file and processing it to finally upload its contents to the [Headless Hathor Wallet API](https://wallet-headless.docs.hathor.network/) site.
+
+# Guide-level explanation
+### JS to JSON conversion
+The platform that manages the OpenAPI Documentation static website, [Redoc](https://github.com/Redocly/redoc), only interacts with `.json` files. So we need to convert our `src/api-docs.js` file into a `.json` one whenever we want to use this platform.
+
+A script should be built to make this conversion on the `scripts` folder, outputting the results to the `tmp/` folder. All following steps will be executed on this converted file. Some specific conditions to execute:
+- Remove the `securitySchemes` from the `components` property
```

Comment:
Why do we need to remove it? And what are they used for if we will remove them? Shouldn't we remove them from the `src/api-docs.js` file?

Reply:
The current contents of this property are:
```json
"securitySchemes": {
  "ApiKeyAuth": {
    "type": "apiKey",
    "in": "header",
    "name": "X-API-KEY"
  }
}
```
So, it's not that we're sharing any sensitive information.
@pedroferreira1 , since you have more experience with this workflow, could you share with us why we currently remove this property before publishing the api-docs?

Reply:
Removal suggested on [a336ca7](https://github.com/HathorNetwork/rfcs/pull/56/commits/a336ca7eefcaa335323fd15b6f1573613196db66)